functions/Get-DbaOperatingSystem.ps1

```diff
@@ -68,37 +68,36 @@ function Get-DbaOperatingSystem {
 			}
 
 			$activePowerPlan = ($powerPlan | Where-Object IsActive).ElementName -join ','
-
-			$inputObject = [PSCustomObject]@{
-				ComputerName = $computerResolved
-				InstanceName = $computer.InstanceName
-				SqlInstance = $computer.SqlInstanceName
-				Manufacturer = $os.Manufacturer
-				Organization = $os.Organization
-				Architecture = $os.OSArchitecture
-				Version = $os.Version
-				Build = $os.BuildNumber
-				InstallDate = [DbaDateTime]$os.InstallDate
-				LastBootTime = [DbaDateTime]$os.LastBootUpTime
-				LocalDateTime = [DbaDateTime]$os.LocalDateTime
-				TimeZone = $tz.Caption
+			$language = Get-Language $os.OSLanguage
```

Comment:
I don't see the purpose in this one but your the boss.

Most folks know what the language id should be for their environment.

As well we could have used `Get-Culture` which is builtin cmdlet for PowerShell 1.0 and up to grab the same information from the machine. Although don't think it includes remote support so would have had to use `Invoke-Command`, but either way you get the same info and the correct OS level (and doesn't require another command to keep up with internally :) [Which `us_english` is SQL Server's spelled out version, the OS uses `en-US` format]

![image](https://user-images.githubusercontent.com/11204251/28621350-451b776a-71d6-11e7-830c-2e79eb01a201.png)